targets/esp8266/source/jerry_run.c

```diff
@@ -45,7 +46,8 @@ int js_entry (const char *source_p, const size_t source_size)
   {
     if ((flags & JERRY_FLAG_PARSE_ONLY) == 0)
     {
-      ret_code = jerry_run ();
+      jerry_api_value_t res;
+      ret_code = jerry_run (&res);
```

Comment:
`jerry_api_release_value` should be called if jerry_run failes. The parameter of `jerry_run` is for error value not for result, so the name of the variable is misleading.